test/test_no_ssl.py

```diff
@@ -65,28 +63,22 @@ def pop(self):
 module_stash = ModuleStash('urllib3')
 
 
-class TestWithoutSSL(unittest.TestCase):
```

Comment:
Surprise this was imported elsewhere in the tests! Can't change the name without also changing those other tests.